bin/utils.sh

```diff
@@ -112,5 +112,31 @@ exclude_core_tests() {
 	done
 }
 
+update_core_tests() {
+	local wp_tests_dir="$1"
+
+	if [ -z "$wp_tests_dir" ]; then
+		echo "WP_TESTS_DIR was not passed in";
+		exit 1;
+	fi
+
+
+	local header_handling_locations=(
+		"$wp_tests_dir/tests/xmlrpc/wp/newComment.php:Tests_XMLRPC_wp_newComment"
+	)
+
+	local header_handling='\/**\n * @runTestsInSeparateProcesses\n * @preserveGlobalState disabled\n *\/'
```

Comment:
I tried to add `@runInSeparateProcess` per each test that needs it, but it hanged forever untill it timed-out. This seems to work better